specification/peering/resource-manager/Microsoft.Peering/preview/2019-09-01-preview/peering.json

```diff
@@ -158,7 +158,7 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/providers/Microsoft.Peering/peerAsns/{peerAsnName}": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Peering/peerAsns/{MyPeerAsn}": {
```

Comment:
I don't think the names in this file should be changed. They are all names of path parameters, not examples.